File: Datasets/TrickyBugs/GenProgs/dpp_generated_progs_python/p03294/p03294_num3_parsed.py
import sys

# Read input
N = int(sys.stdin.readline().strip())
a = list(map(int, sys.stdin.readline().split()))

# Calculate maximum value of f
max_f = 0
for m in range(max(a)):
    f = sum(m % ai for ai in a)
    max_f = max(max_f, f)

# Print the result
print(max_f)

File: Datasets/TrickyBugs/GenProgs/dpp_generated_progs_cpp/p03294/p03294_num3_parsed.cpp
#include <iostream>
#include <vector>
#include <algorithm>

int main() {
    int N;
    std::cin >> N;
    
    std::vector<int> a(N);
    for (int i = 0; i < N; i++) {
        std::cin >> a[i];
    }
    
    int max_f = 0;
    for (int m = 0; m < 100000; m++) {
        int f = 0;
        for (int i = 0; i < N; i++) {
            f += m % a[i];
        }
        max_f = std::max(max_f, f);
    }
    
    std::cout << max_f << std::endl;
    
    return 0;
}